src/App.tsx

```diff
@@ -24,14 +24,10 @@ THE SOFTWARE.
 
 import React, { useContext } from "react";
 import { useEffect, useState } from "react";
-import {
-  EuiCode,
-  EuiEmptyPrompt,
-  EuiThemeProvider,
-  EuiThemeAmsterdam,
-} from "@elastic/eui";
+import { EuiCode, EuiEmptyPrompt, EuiProvider } from "@elastic/eui";
+import createCache from "@emotion/cache";
```

Comment:
I don't see the @emotion/cache on the deps list.

Reply:
We already had it as a dep of `@emotion/react`, which I guess is why it worked. I've added it as a direct dep. d8e107d7f2e77f14e8ae730e55a06162befeb814